pulser-core/pulser/sequence/_seq_drawer.py

```diff
@@ -44,100 +46,107 @@
 
 @dataclass
 class ChannelDrawContent:
-    """The contents for drawingflake a single channel."""
+    """The contents for drawing a single channel."""
 
-    time: list[int]
-    amplitude: list[float]
-    detuning: list[float]
-    phase: list[float]
+    samples: ChannelSamples
     target: dict[Union[str, tuple[int, int]], Any]
-    measurement: Optional[str] = None
     interp_pts: dict[str, list[list[float]]] = field(default_factory=dict)
 
     def __post_init__(self) -> None:
+        self.samples.phase = self.samples.phase / (2 * np.pi)
+        self._samples_from_curves = {
+            "amplitude": "amp",
+            "detuning": "det",
+            "phase": "phase",
```

Comment:
Very sad to have different names across the code base 😢

Reply:
I agree, but I think it's not worth changing now

Reply:
Definitely out of scope for this PR indeed